### ISSUE TITLE
Will Steam drivers still preform worst compared to oculus drivers. 

### ISSUE THREAD

**Issue by gme1999** [OpenOculus/OculusKiller#67]
I understand the whole point of this is to remove the dash for increased performance, thus removing the ability to play oculus native games. But one thing I noticed is that oculus games always run better compared to steam games, probably because oculus devices are made for oculus drivers. so my question is will this configuration allow steam games to run as good as native oculus games, does this mod make the steam drivers more compatable or does it just remove the dash?

**Comment by HamzaETTH** [comment]
I don't think so, according to the [OVR Toolkit dev](https://store.steampowered.com/app/1068820/OVR_Toolkit/):

> Running games through SteamVR on an Oculus headset may decrease performance, this is not an issue specific to OVR Toolkit. This is due to Oculus rendering the same image twice, once when SteamVR compositor renders the output, then again when Oculus Runtime renders the output. Unfortunately, this is something only Oculus/Facebook can fix!
